externs/shaka/player.js

```diff
@@ -299,6 +300,8 @@ shaka.extern.BufferedInfo;
  *   The video pixel aspect ratio provided in the manifest, if present.
  * @property {?string} hdr
  *   The video HDR provided in the manifest, if present.
+ * @property {?string} hdr
```

Comment:
typo: colorGamut

This `@property` annotation only affects the docs, not the compiler, so you're able to pass lint/compile checks with this typo.